bqplot/marks.py

```diff
@@ -522,8 +522,9 @@ class _ScatterBase(Mark):
         'opacity': {'dimension': 'opacity'},
         'rotation': {'dimension': 'rotation'}
     }).tag(sync=True)
-    default_opacities = List(trait=Float(1.0, min=0, max=1, allow_none=True))\
-        .tag(sync=True, display_name='Opacities')
+    default_opacities = Array(None, allow_none=True)\
```

Comment:
I'm not entirely sure, but I think this change breaks the `default_opacities` in the default Scatter implementation

Reply:
Fixed c167c2a